eth_portal/web3_encoding.py

```diff
@@ -57,7 +52,6 @@ def receipt_content_value(receipts) -> bytes:
     """
     # Awkward quirk that ssz must take an iterable of individual bytes, instead of `bytes`
     quirky_byte_list = [
-        [bytes([byte]) for byte in receipt.encode()]
-        for receipt in receipts
+        [bytes([byte]) for byte in receipt.encode()] for receipt in receipts
```

Comment:
This line I liked a bit better before `black`, but not quite enough to turn off formatting. Also, when py-ssz is updated, it will just read as:
```py
        receipt.encode() for receipt in receipts
```